README.md

```diff
@@ -147,7 +147,7 @@ command:
 Required settings:
 
 * `JANKY_BASE_URL`: The application URL **with** a trailing slash. Example:
-  `http://mf-doom-42.herokuapp.com/`.
+  ``.
```

Comment:
Would you mind removing the two backticks and the word `Example:` above as well? Since there's no actual app at that URL, then I'd rather see the example bits go away completely. Thanks!